blank-canvas/style.css

```diff
@@ -47,9 +39,15 @@ GNU General Public License for more details.
 
 /* Add some top padding if the first block on the page is not full-width or a spacer. */
 
-.single .entry-content > *:first-child:not(.alignfull):not(.wp-block-spacer),
-.page .entry-content > *:first-child:not(.alignfull):not(.wp-block-spacer) {
-	padding-top: var(--global--spacing-vertical);
+.single .entry-content > .wp-block-image.alignfull:first-child,
```

Comment:
Could this just be a single `.entry-content > ...` rule, or is it not specific enough?

Reply:
I also ask because the specificity of these rules makes it so they do not apply to posts — is that the intention?

Reply:
Yeah, it _should_ apply to individual posts and pages, but not posts within a feed. So that's why the extra specificity is necessary: `.single` is for single posts, `.page` is for single pages.